src/Nethermind/Nethermind.Config/ConfigSourceHelper.cs

```diff
@@ -51,7 +51,13 @@ public static object ParseValue(Type valueType, string valueString)
                     var i = 0;
                     foreach (var valueItem in valueItems)
                     {
-                        var itemValue = GetValue(itemType, valueItem);
+                        string item = valueItem;
+                        if (valueItem.StartsWith('"') && valueItem.EndsWith('"'))
+                        {
+                            item = valueItem.Substring(1, valueItem.Length - 2);
```

Comment:
Trim("\"")?

Reply:
Sure it does not break anything?

Reply:
1. It does not break anything, we didn't have string[] in configs yet.
2. Trim(""") would trim multiple characters, we just want to trim one from start and end if present.